pythonforandroid/recipes/opencv/__init__.py

```diff
@@ -17,7 +17,7 @@ class OpenCVRecipe(NDKRecipe):
     '''
     version = '4.0.1'
     url = 'https://github.com/opencv/opencv/archive/{version}.zip'
-    depends = ['numpy']
+    depends = ['numpy', ('python2', 'python3', 'python3crystax')]
```

Comment:
I think you can drop this change too for the same reason @inclement gave on the other